src/Admin/Extension/TreeAdminExtension.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Zicht\Bundle\MenuBundle\Admin\Extension;
+
+use Sonata\AdminBundle\Admin\AbstractAdminExtension;
+use Sonata\AdminBundle\Form\FormMapper;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+
+class TreeAdminExtension extends AbstractAdminExtension
+{
+    /**
+     * {@inheritDoc}
+     */
+    public function configureFormFields(FormMapper $formMapper)
+    {
+        $subject = $formMapper->getAdmin()->getSubject();
+        // follow the path from Zicht\Bundle\AdminBundle\Admin\TreeAdmin
+        $formMapper
+            ->tab('General')
+            ->with('General')
+                ->add('language', TextType::class, ['required' => false, 'attr' => ['placeholder' => 'nl']])
```

Comment:
Zou hier betere de LanguageType kunnen gebruiken van de zicht/translations-bundle, zie https://github.com/zicht/translations-bundle/blob/release/5.x/src/Zicht/Bundle/TranslationsBundle/Form/Type/LanguageType.php

Dan is het hetzelfde als andere plaatsen waar we een taal config aanbieden

Reply:
Moet je die ook requiren in de composer.json
\+ als deze niet in het project zit, dan is er geen configuratie, dus geen te selecteren opties
redelijk breaking change dan

Reply:
Die Translations-bundle wordt niet required door deze bundle, die is nog lang niet overal aanwezig. Voor een `4.0.1` wijziging zie ik dat niet zo zitten, dit PR is meer een `quick win` zodat ik niet zo'n migratie hoef te schrijven zoals hier https://github.com/zicht/concertgebouworkest.nl/pull/469 wat ook weer allemaal moeilijkheden met zich meebrengt.

Reply:
Check, breaking change is hier idd niet handig.

Kunnen het ook compatible toevoegen door te checken of de LanguageType bestaat en dan die te gebruiken... dat zou een goede oplossing zijn wat mij betreft

Reply:
Ik kijk morgen even wat daarvoor haalbaar is!

Reply:
Dit moet dan nog wachten tot de translations-bundle compatible is met Symfony 4.

Reply:
> Dit moet dan nog wachten tot de translations-bundle compatible is met Symfony 4.

Dat dacht ik inderdaad al, dat dat ook nog niet was gedaan

Reply:
In dat geval: Approved